File: src/main/java/com/mynimef/bot/containers/ButtonLine.java
package com.mynimef.bot.containers;

import java.util.ArrayList;
import java.util.List;

public class ButtonLine {

    private final List<Button> line;

    public ButtonLine(Button button) {
        this.line = new ArrayList<>();
        this.line.add(button);
    }

    public List<Button> getLine() { return line; }

    public void addButton(Button button) {
        this.line.add(button);
    }

}

File: src/main/java/com/mynimef/bot/containers/ButtonLine.kt
package com.mynimef.bot.containers

class ButtonLine(button: Button) {

    private val _line: MutableList<Button> = ArrayList<Button>().apply {
        add(button)
    }
    val line: List<Button> = _line

    fun addButton(button: Button) {
        _line.add(button)
    }

}